nixos/modules/services/web-apps/peertube.nix

```
@@ -488,6 +488,11 @@ in {
 
     services.nginx = lib.mkIf cfg.configureNginx {
       enable = true;
+      upstreams = {
+        "backend-peertube" = {
+          servers = { "127.0.0.1:${toString cfg.listenHttp}" = { fail_timeout = "0"; }; };
+        };
+      };
```

```suggestion
      upstreams."backend-peertube".servers = {
        "127.0.0.1:${toString cfg.listenHttp}".fail_timeout = "0";
      };
```

nit

not sure if we should set keepalive here

---

Recommended if using one server.

---

I've just slightly changed the formatting to be more compact :)

---

Ok :)